go.mod

```diff
@@ -1,29 +1,30 @@
 module github.com/openshift/run-once-duration-override-operator
 
-go 1.22.0
+go 1.23.0
 
-toolchain go1.22.1
+toolchain go1.23.4
```

Comment:
Assume that this is required?

Reply:
I experience the same thing. This is automatic.

Reply:
Yeah it is, I run the go mod tidy in the container. Thanks for the tags!